packages/perspective/test/js/leaks.spec.js

```diff
@@ -111,6 +122,19 @@ test.describe("leaks", function () {
             view.delete();
             table.delete();
         });
+
+        test.skip("csv loading does not leak", async () => {
```

Comment:
Just double checking, should this be skipped?

Reply:
It takes forever to run is the only reason to skip it